test/gtest/ucp/test_ucp_tag_match.cc

```diff
@@ -18,7 +18,7 @@ class test_ucp_tag_match : public test_ucp_tag {
     {
         m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_ENABLE", "y"));
         if (RUNNING_ON_VALGRIND) {
-            m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_MAX_BCOPY", "8k"));
+            m_env.push_back(new ucs::scoped_setenv("UCX_RC_TM_SEG_SIZE", "8k"));
```

Comment:
you introduced TM_MAX_SIZE name, not TM_SEG_SIZE

Reply:
I was wrong with the name of the new introduced attribute